eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/RecordId.scala

```diff
@@ -0,0 +1,24 @@
+package com.evolutiongaming.kafka.journal.eventual.cassandra
+
+import cats.Monad
+import cats.effect.std.UUIDGen
+import cats.syntax.all.*
+import com.evolutiongaming.scassandra.*
+
+final case class RecordId(value: String) extends AnyVal {
+  override def toString: String = value
+}
+
+object RecordId {
+  def unsafe(value: String): RecordId = new RecordId(value)
+
+  def of[F[_]: Monad: UUIDGen]: F[RecordId] =
```

Comment:
I'm not sure about the naming style in this project but I think `of` shouldn't result in new unique values generated - I'd renamed to smth like `generate` or `new` so it is visible in the code where new values are generated

Reply:
I never have seen such notation within KJ thus no, I will keep `of`